xtask/codegen/src/generate_configuration.rs

```diff
@@ -488,16 +487,16 @@ fn generate_struct(group: &str, rules: &BTreeMap<&'static str, RuleMetadata>) ->
                 matches!(self.recommended, Some(true))
             }
 
-            pub(crate) const fn is_not_recommended(&self) -> bool {
-                matches!(self.recommended, Some(false))
+            pub(crate) const fn is_recommended_unset(&self) -> bool {
```

Comment:
Thanks for that! Naming is hard, and it's these little touches that can make a difference.

Reply:
I agree with you but here is a different case. `is_not_recommended` tests `Some(false)` while `is_recommended_unset` tests `None`. To implement the logic I created the latter one, and the former one is no longer needed in the code, hence the change.

To make the namings more consistent I probably should've renamed `is_recommended` to `is_recommended_true`. But as Ema suggests these preset options will be dropped in v2.0 so I didn't touch them.